cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -86,7 +86,7 @@ const (
 )
 
 // NewControllerManagerCommand creates a *cobra.Command object with default parameters
-func NewControllerManagerCommand() *cobra.Command {
+func NewControllerManagerCommand(stopCh <-chan struct{}) *cobra.Command {
```

Comment:
commands are no longer accepting `stopCh` chan, instead, they register handlers for the signals when they need it - see: https://github.com/kubernetes/kubernetes/pull/76659